Specs/DateTime/English/DateTimeModelComplexCalendar.json

```diff
@@ -2054,74 +2054,80 @@
       "ReferenceDateTime": "2018-05-24T00:00:00"
     },
     "NotSupported": "python",
-    "Results": [{
-      "Text": "april 2017",
-      "Start": 12,
-      "End": 21,
-      "TypeName": "datetimeV2.daterange",
-      "Resolution": {
-        "values": [
-          {
-            "timex": "2017-04",
-            "type": "daterange",
-            "start": "2017-04-01",
-            "end": "2017-05-01"
-          }
-        ]
-      }
-    }]
+    "Results": [
```

Comment:
Are these all just linebreak changes? The format in the repo should be all unix-style line breaks.
Perhaps we need a cleanup PR to standardize all files?

Reply:
Yes, I agree.